app/renderer/components/preferences/payment/ledgerBackup.js

```diff
@@ -0,0 +1,128 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const React = require('react')
+const {StyleSheet, css} = require('aphrodite')
+
+// components
+const ImmutableComponent = require('../../../../../js/components/immutableComponent')
+const Button = require('../../../../../js/components/button')
+
+// style
+const globalStyles = require('../../styles/global')
+const commonStyles = require('../../styles/commonStyles')
+const {paymentCommon} = require('../../styles/payment')
+
+// other
+const aboutActions = require('../../../../../js/about/aboutActions')
+
+class LedgerBackupContent extends ImmutableComponent {
+  copyToClipboard (text) {
+    aboutActions.setClipboard(text)
+  }
+
+  render () {
+    const paymentId = this.props.ledgerData.get('paymentId')
+    const passphrase = this.props.ledgerData.get('passphrase')
+
+    return <div className={css(paymentCommon.board)}>
+      <div className={css(paymentCommon.panel, styles.ledgerBackupContent)}>
+        <span data-l10n-id='ledgerBackupContent' />
+        <div className={css(styles.copyKeyContainer)}>
+          <Button l10nId='copy'
+            className={css(commonStyles.whiteButton, styles.copyButton)}
+            onClick={this.copyToClipboard.bind(this, paymentId)}
+          />
+          <div className={css(styles.keyContainer)}>
+            <h3 className={css(styles.keyContainer__h3)} data-l10n-id='firstKey' />
+            <span className={css(styles.keyContainer__span)}>{paymentId}</span>
+          </div>
+        </div>
+        <div className={css(styles.copyKeyContainer)}>
+          <Button l10nId='copy'
+            className={css(commonStyles.whiteButton, styles.copyButton)}
+            onClick={this.copyToClipboard.bind(this, passphrase)}
+          />
+          <div className={css(styles.keyContainer)}>
+            <h3 className={css(styles.keyContainer__h3)} data-l10n-id='secondKey' />
+            <span className={css(styles.keyContainer__span)}>{passphrase}</span>
```

Comment:
STR:
1. Clear profile
2. Enable payments
3. Go to advanced settings
4. Click backup your wallet
5. Key 2 doesn't exist
6. Try to copy key 2